### ISSUE TITLE
Database corrupts when changing tags

### ISSUE THREAD

**Issue by WatermelonDriveway** [ncmpcpp/ncmpcpp#203]
If I change some metadata info on some albums or rename some folders, I will get ncmpcpp connection timeouts.  The fix I've been doing is just doing 'rm mpd.db' in my .mpd directory, and then touching another mpd.db.  When I run mpd again, I get a database corrupted message, but it will still work and then I can update the database to get all my albums back.

Am I doing something wrong here?  This process takes a few minutes and is pretty inconvenient

**Comment by arybczak** [comment]
When update is issued, all ncmpcpp does is sending the update command to MPD, so I can't help you here.

**Comment by arybczak** [comment]
Also, you shouldn't supposed to be doing any of this with the db file.

**Comment by WatermelonDriveway** [comment]
When I update my database with 'u'.  It says "connection refused" on the bottom of the window

**Comment by arybczak** [comment]
What does MPD log say?

**Comment by arybczak** [comment]
Database is automatically created, there is no need to do that manually.

**Comment by WatermelonDriveway** [comment]
Doesn't look like I have a log file, I'll have to update you next time this happens to me.  I just reinstalled mpd and got a no database error: 'exception: Failed to open /Users/Jnny/.mpd/mpd.db: No such file or directory', but it ran anyway showing my library without creating a db file.  When I pressed 'u' it created one for me.  Hopefully this will solve the problem.  I suppose I can go into IRC if it happens again and try to find you or somebody else if you want to close the issue.  Thanks!
